x-pack/platform/plugins/shared/logs_shared/kibana.jsonc

```diff
@@ -31,7 +31,8 @@
     ],
     "requiredBundles": [
       "kibanaUtils",
-      "kibanaReact"
+      "kibanaReact",
+      "discover",
```

Comment:
No need to add discover as a dependency if you don't plan on using any of its services. Having it on the `tsconfig.json` is enough IMO.

Reply:
It's already there but without this change I got a [failing build](https://buildkite.com/elastic/kibana-pull-request/builds/277562#01951d74-1238-440b-9e9b-2d9f585140fc), what am I missing?

Reply:
mmm 🤔 
yeah, I don't like that approach, whenever the params are updated you won't have the last version
I will revert importing `DISCOVER_APP_LOCATOR` to check if I can get rid of the dependency